packages/zowe-explorer/src/command/TsoCommandHandler.ts

```diff
@@ -342,6 +330,23 @@ export class TsoCommandHandler extends ZoweCommandProvider {
                     );
                 }
             }
+        } else {
+            const profileManager = await Profiles.getInstance().getCliProfileManager("tso");
+            if (profileManager) {
+                try {
+                    const profiles = await profileManager.loadAll();
+                    for (const item of profiles) {
+                        if (item.type === "tso") {
+                            tsoProfiles.push(item);
+                        }
+                    }
+                    tsoProfile = await this.selectTsoProfile(tsoProfiles);
+                } catch (error) {
+                    if (!error?.message?.includes(`No default profile set for type "tso"`)) {
+                        vscode.window.showInformationMessage(error);
+                    }
+                }
+            }
```

Comment:
I like the logic switch, that will prevent trying to create a profileManager when using teamConfig 
🥳